tycho-metadata-model/src/main/java/org/eclipse/tycho/model/classpath/JUnitClasspathContainerEntry.java

```diff
@@ -29,8 +29,8 @@ public interface JUnitClasspathContainerEntry extends ClasspathContainerEntry {
     static final JUnitBundle JUNIT4_PLUGIN = JUnitBundle.of("org.junit", "[4.13.0,5.0.0)",
             "org.apache.servicemix.bundles", "org.apache.servicemix.bundles.junit");
 
-    static final JUnitBundle HAMCREST_CORE_PLUGIN = JUnitBundle.of("org.hamcrest.core", "[1.1.0,2.0.0)",
-            "org.apache.servicemix.bundles", "org.apache.servicemix.bundles.hamcrest");
+    static final JUnitBundle HAMCREST_CORE_PLUGIN = JUnitBundle.of("org.hamcrest", "[2.0.0,3.0.0)", "org.hamcrest",
```

Comment:
the specified maven artifact is not a bundle!

Reply:
Hmm, this looks like a bundle to me: 
```
Manifest-Version: 1.0
Implementation-Title: hamcrest
Automatic-Module-Name: org.hamcrest
Bundle-SymbolicName: org.hamcrest
Implementation-Version: 2.2
Bundle-ManifestVersion: 2
Bnd-LastModified: 1571259132000
Import-Package: javax.xml.namespace;resolution:=optional,javax.xml.xpa
 th;resolution:=optional,org.w3c.dom;resolution:=optional,org.hamcrest
 ;version="[2.2,3)",org.hamcrest.beans;version="[2.2,3)",org.hamcrest.
 collection;version="[2.2,3)",org.hamcrest.comparator;version="[2.2,3)
 ",org.hamcrest.core;version="[2.2,3)",org.hamcrest.internal;version="
 [2.2,3)",org.hamcrest.number;version="[2.2,3)",org.hamcrest.object;ve
 rsion="[2.2,3)",org.hamcrest.text;version="[2.2,3)",org.hamcrest.xml;
 version="[2.2,3)"
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
Tool: Bnd-3.4.0.201707252008
Implementation-Vendor: hamcrest.org
Export-Package: org.hamcrest;version="2.2";uses:="javax.xml.namespace,
 org.hamcrest.collection,org.hamcrest.core,org.hamcrest.internal,org.w
 3c.dom",org.hamcrest.beans;version="2.2";uses:="org.hamcrest",org.ham
 crest.collection;version="2.2";uses:="org.hamcrest",org.hamcrest.comp
 arator;version="2.2";uses:="org.hamcrest",org.hamcrest.core;version="
 2.2";uses:="org.hamcrest",org.hamcrest.internal;version="2.2";uses:="
 org.hamcrest",org.hamcrest.io;version="2.2";uses:="org.hamcrest",org.
 hamcrest.number;version="2.2";uses:="org.hamcrest",org.hamcrest.objec
 t;version="2.2";uses:="org.hamcrest",org.hamcrest.text;version="2.2";
 uses:="org.hamcrest",org.hamcrest.xml;version="2.2";uses:="javax.xml.
 namespace,org.hamcrest,org.w3c.dom"
Bundle-Name: hamcrest
Bundle-Version: 2.2
Created-By: 1.8.0_232-ea (Private Build)
```
What issue do you have with it ?

Reply:
hm... it seems the `hamcrest` vs `hamcrest-core` has tricked me here :-\

So everything looks fine here :+1: